### PR TITLE
Add WPF hosted mode

### PR DESCRIPTION
**Pull request by jeromelaban**
## PR Type
- Feature

This PR adds the ability to run the WebAssembly-scoped netstandard2.0 code of Uno.UI inside of the WPF host to allow for easier debugging.

### CODE REVIEW

File: src/Uno.UWP/UI/Core/CoreDispatcher.netstd.cs [hunk 1]
```diff
@@ -10,9 +10,17 @@ public sealed partial class CoreDispatcher
     {
 		private Timer _timer;
 
+		/// <summary>
+		/// Provide a action that will delegate the dispach of CoreDispatcher work
+		/// </summary>
+		public static Action<Action> DispatchOverride;
```
Comment (davidjohnoliver) [inline src/Uno.UWP/UI/Core/CoreDispatcher.netstd.cs]: hmm... maybe this should at least be marked EditorBrowsable never

**Comment by davidjohnoliver** [comment]
Interesting! Does the JS layer work the same way as on WASM?

**Comment by jeromelaban** [comment]
It should work the same way yes, with a little catch about the async dispatch, but not sure yet how this is going to affect the behavior.

File: src/Uno.UWP/UI/Core/CoreDispatcher.netstd.cs (hunk 1 above)
Comment (jeromelaban) [reply src/Uno.UWP/UI/Core/CoreDispatcher.netstd.cs]: Sure, I'll add that.

**Pull request merged**